docs/superpowers/specs/2026-04-28-easy-policy-interpretation-design.md

```
@@ -0,0 +1,381 @@
+applyStart/applyEnd: ...
+
+[원문 - 구조화 필드]
+summary:           ...
+body:              ...
+supportTarget:     ...
+selectionCriteria: ...
+supportContent:    ...
+applyMethods:      ...
+
+[원문 - 보강(있으면)]
+PolicyDocument 청크 1: ...
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)